scripts/benchmark-tauri-b1.py

```
@@ -57,7 +57,7 @@ def command_text(parts: list[str | Path]) -> str:
 
 
 def run_checked(parts: list[str | Path], *, timeout: float = 1200) -> str:
-    result = subprocess.run(
+    result = subprocess.run(  # nosec -- nosemgrep: python.lang.security.audit.dangerous-subprocess-use-audit -- argv is a list (no shell), and the command name is always a literal; only internally-generated identifiers (uuid4/getpid/a fixed scenario tuple) vary.
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)